{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -42,19 +43,16 @@ repository = "https://github.com/Nieuwe-Warmte-Nu/{{cookiecutter.repo_name}}"
 changelog = "https://github.com/Nieuwe-Warmte-Nu/{{cookiecutter.repo_name}}/blob/main/CHANGELOG.md"
 
 [project.scripts]
-newchess = "{{cookiecutter.python_import_name}}.infrastructure.app:cmdline_app"
+{{cookiecutter.python_import_name}} = "{{cookiecutter.python_import_name}}.infrastructure.app:cmdline_app"
```

Comment:
Should be 
`{{cookiecutter.python_import_name}} = "{{cookiecutter.python_import_name}}:main"`

I wanted to keep the project.scripts table in here as it's something we dont do that often (and thus the user will need to dive into documentation to figure out how it works)

Reply:
Yep, will change.